src/systems/diff_drive/DiffDrive.cc

```diff
@@ -271,6 +274,9 @@ void DiffDrive::Configure(const Entity &_entity,
   this->dataPtr->odomPub = this->dataPtr->node.Advertise<msgs::Odometry>(
       odomTopic);
 
+  this->dataPtr->tfPub = this->dataPtr->node.Advertise<msgs::Pose_V>(
+      "tf");
```

Comment:
Can we prefix this topic with the model name by default?

It would also be nice to make it configurable from SDF like `odom_topic` is.